Assets/Scripts/Crow/CrowManager.cs

```diff
@@ -35,6 +35,8 @@ public class CrowManager : MonoBehaviour
     private const int startXPos = 16; //Offscreen X location 'ahead' of the player
     private const int endXPos = -10; //Offscreen X location 'behind' the player
     private int zPos = 0; //Initial z-axis location before updating with random value
+    private const int minZPos = 1; //Boundary Z-pos at right side of screen
```

Comment:
I should've added a more coherent way to get the z values for the boundaries in the tile manager so I get why this is hardcoded for now.